board/nrf52832_thingy_52/board.h

```diff
@@ -0,0 +1,92 @@
+/* Copyright 2020 Wirepas Ltd. All Rights Reserved.
+ *
+ * See file LICENSE.txt for full license details.
+ *
+ */
+
+/**
+ * @file
+ *
+ * Board definition for the
+ * <a href="https://www.nordicsemi.com/Products/Development-hardware/Nordic-Thingy-52"> Nordic-Thingy-52</a>
+ */
+
+#ifndef _BOARD_NRF52832_THINGY_52_BOARD_H_
```

Comment:
Please stick to the naming scheme of the other boards and use `pcaXXX` style of names. 
According to the board i just stole from your desk the thingy is called `pca20020`

Reply:
Done